src/validation.cpp

```diff
@@ -2867,7 +2875,8 @@ bool CChainState::ActivateBestChain(BlockValidationState &state, const CChainPar
         LimitValidationInterfaceQueue();
 
         {
-            LOCK2(cs_main, ::mempool.cs); // Lock transaction pool for at least as long as it takes for connectTrace to be consumed
+            LOCK(cs_main);
```

Comment:
nit:
```suggestion
            LOCK(::cs_main);
```

Reply:
Thanks. Will add the `::` when I have to retouch or on the next commit that touches those lines.